packages/pilot/src/components/Event/style.css

```
@@ -164,3 +164,37 @@
 .active:not(:only-child):after {
   background-color: var(--color-white);
 }
+
+@media print {
+
+  .event {
+    align-items: baseline;
+    background: none;
+    display: flex;
+    padding: 0;
+    width: 100%;
+  }
+
+  .title:before {
+    content: "-";
+  }
+
+  .header {
+    padding-top: 0 !important; /* stylelint-disable-line declaration-no-important */
```

Este `!important` pode ser substituido por uma especificação melhor sob a classe acima ex:
```suggestion
    .event {
    ...
      & > .header,
      &.active + .event > .header {
        padding-top: 0;
      }
    }
```